Prefabs/Locomotion/Movement/MovementAmplifier/SharedResources/Scripts/MovementAmplifierFacade.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace VRTK.Core.Prefabs.Locomotion.Movement.MovementAmplifier
+{
+    using UnityEngine;
+    using VRTK.Core.Data.Attribute;
+
+    /// <summary>
+    /// The public interface for the MovementAmplifier prefab.
+    /// </summary>
+    public class MovementAmplifierFacade : MonoBehaviour
+    {
+        #region Facade Settings
```

Comment:
Generally the `Facade Settings` are located in the `Internal Setup` script where the `Facade` is injected into the Internal components.

I've noticed a couple of the other prefabs (BodyRepresentation and Climb) have this `Facade Settings` thing, but I think they should also be updated to follow the remaining prefabs pattern.

This can be renamed to a single group like `Amplifier Settings`

or maybe create a number of headers like

`Tracking Settings`
source/target

`Movement Settings`
ignored Radius/ Multipler

(or whatever works best)

Reply:
👍 I chose the latter suggestion (two headers).